src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -11,19 +12,34 @@ import { AcqCheckboxFilter } from '@folio/stripes-acq-components';
 
 import { ListSelect } from './ListSelect/ListSelect';
 import { ListFileUploader } from './ListFileUploader/ListFileUploader';
-import { buildCheckboxFilterOptions } from './utils';
+import { buildCheckboxFilterOptions } from './utils/optionsRecordIdentifiers';
 import { EDIT_CAPABILITIES } from '../../../constants/optionsRecordIdentifiers';
+import { getFileInfo } from './utils/getFileInfo';
 
-export const BulkEditListFilters = () => {
+export const BulkEditListFilters = (
+  {
+    setFileUploadedName,
+    setIsFileUploaded,
+    isFileUploaded,
+  },
+) => {
   const [criteria, setCriteria] = useState('identifier');
   const [isLoading, setLoading] = useState(false);
   const [isDropZoneActive, setDropZoneActive] = useState(false);
+  const [fileExtensionModalOpen, setFileExtensionModalOpen] = useState(false);
   const [filters, setFilter] = useState({
     capabilities: ['users'],
+    recordIdentifier: '',
   });
+  const [isDropZoneDisabled, setIsDropZoneDisabled] = useState(true);
 
   const capabilitiesFilterOptions = buildCheckboxFilterOptions(EDIT_CAPABILITIES);
 
+  useEffect(() => {
+    if (isFileUploaded || filters.recordIdentifier === '') {
```

Comment:
Can be simplified with `(isFileUploaded || !filters.recordIdentifier)`

Reply:
I have fixed it already!